.github/workflows/pages.yaml

```diff
@@ -3,8 +3,6 @@ on:
   push:
     branches:
       - main
-  pull_request:
-  workflow_dispatch:
```

Comment:
workflow_dispatch is usually nice to have to be able to just rerun the action if something broke or so.

Reply:
you can just retry the last done

Reply:
it's not like with the rebase action of nuschtpkgs

Reply:
also true 🤔